index.html

```diff
@@ -1,108 +1,280 @@
-        <h2 class="dev-tag">Software Developer</h2>
-        <p class="first-paragraph">I can help you build a product, feature or website<br> Look through some of my work
-            and experience!If <br>
-            you like what you see and have a project you need<br> coded, don’t hestiate to contact me.</p>
+      <h1 class="name">Marlyn Mayienga</h1>
+      <img class="first-img" src="./circle_header.svg" alt="circle 2 header"/>
+      <h2 class="dev-tag">Software Developer</h2>
+      <p class="first-paragraph">
+        I can help you build a product, feature or website. Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
 
-        <div class="divider"></div>
-        <!-- TECHNLOGIES -->
-        <section class="technologies">
-            <div class="technologies-item">
-                <h2 class="technologies-title">Languages</h2>
-                <ul>
-                    <li>JavaScript</li>
-                    <li>Ruby</li>
-                    <li>HTML</li>
-                    <li>CSS</li>
-                </ul>
-            </div>
-            <div>
-                <div class="technologies-item">
-                    <div class="technologies-title">Frameworks</div>
-                    <ul>
-                        <li>BootStrap</li>
-                        <li>Ruby on Rails</li>
-                        <li>RPSec</li>
-                        <li>CapyBase</li>
-                        <li>Selenium</li>
-                    </ul>
-                </div>
-            </div>
-            <div>
-                <div class="technologies-item">
-                    <div class="technologies-title">Skills</div>
-                    <ul>
-                        <li>Codekit</li>
-                        <li>GitHub</li>
-                        <li>CodePEn</li>
-                        <li>GitLab</li>
-                        <li>Terminal</li>
-                    </ul>
-                </div>
+      <div class="divider"></div>
+      <!-- TECHNLOGIES -->
+      <section class="technologies">
+        <div class="technologies-item">
+          <h2 class="technologies-title">Languages</h2>
+          <ul>
+            <li>JavaScript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div>
+          <div class="technologies-item">
+            <div class="technologies-title">Frameworks</div>
+            <ul>
+              <li>BootStrap</li>
+              <li>Ruby on Rails</li>
+              <li>RPSec</li>
+              <li>CapyBase</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+        </div>
+        <div>
+          <div class="technologies-item">
+            <div class="technologies-title">Skills</div>
+            <ul>
+              <li>Codekit</li>
```

Comment:
- [ ] every separate list has a header and all headers should have the same styling as you can see in the below print screen this is not the case
![Screenshot 2022-04-24 000729](https://user-images.githubusercontent.com/39240820/164947483-0e9525b4-4c67-4db6-8117-0735cb72fe79.png)
.